WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -364,7 +366,11 @@ class WPMainActivityViewModel @Inject constructor(
         val followSiteTask = quickStartRepository.quickStartType
```

Comment:
Nitpick: I think we can remove the comment on line 365. 
```
// For now, we only do this for the FOLLOW_SITE task.
```

Reply:
Removed in 49d11a1.